Tests/SimpleNetworkServiceTests/SimpleNetworkServiceTests.swift

```diff
@@ -0,0 +1,8 @@
+import XCTest
+import SimpleNetworkService
```

Comment:
nit:

You can sort imports alphabetically.

Reply:
ok

Reply:
fixed